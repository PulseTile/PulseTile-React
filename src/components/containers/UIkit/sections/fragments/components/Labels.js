import React from 'react';

/**
 * This component returns Labels section for COMPONENTS
 */
const Labels = () => {
  return (
    <div id="labels" className="ui-section">
      <strong className="ui-title">Labels</strong>
      <div className="form-group-wrapper">
        <div className="form-group">
          <div className="form-control-static">
            <div className="control-group-wrapper">
              <div className="control-group with-indent">
                <span className="label label-default">label default</span>
                <span className="label label-primary">label primary</span>
                <span className="label label-success">label success</span>
                <span className="label label-info">label info</span>
                <span className="label label-warning">label warning</span>
                <span className="label label-danger">label danger</span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Labels;
