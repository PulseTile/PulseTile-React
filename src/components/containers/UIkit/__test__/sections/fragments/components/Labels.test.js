import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Labels from '../../../../sections/fragments/components/Labels';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Labels />', () => {
  it('should renders Labels with props correctly', () => {
    const component = shallow(<Labels />);
    expect(component).toMatchSnapshot();
  });
});
