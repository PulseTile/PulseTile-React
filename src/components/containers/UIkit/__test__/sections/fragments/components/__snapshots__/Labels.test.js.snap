// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Labels /> should renders Labels with props correctly 1`] = `
<div
  className="ui-section"
  id="labels"
>
  <strong
    className="ui-title"
  >
    Labels
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="form-control-static"
      >
        <div
          className="control-group-wrapper"
        >
          <div
            className="control-group with-indent"
          >
            <span
              className="label label-default"
            >
              label default
            </span>
            <span
              className="label label-primary"
            >
              label primary
            </span>
            <span
              className="label label-success"
            >
              label success
            </span>
            <span
              className="label label-info"
            >
              label info
            </span>
            <span
              className="label label-warning"
            >
              label warning
            </span>
            <span
              className="label label-danger"
            >
              label danger
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
